src/internal/m365/backup.go

```diff
@@ -156,6 +156,17 @@ func (ctrl *Controller) IsBackupRunnable(
 	service path.ServiceType,
 	resourceOwner string,
 ) (bool, error) {
+	if service == path.GroupsService {
+		_, err := ctrl.AC.Groups().GetByID(ctx, resourceOwner)
+		if err != nil {
+			// TODO(meain): check for error message in case groups are
+			// not enabled at all similar to sharepoint
+			return false, err
+		}
+
+		return true, nil
```

Comment:
todo here: you'll need to check that the group [IsTeam](https://github.com/alcionai/corso/pull/4053/files#) here.  Currently, if it's not a team, then we have nothing to backup.

Reply:
IIUC, groups have sites associated with them even when it is not a team.

Reply:
Security groups does not seem to have a SP site, but other groups do.
![2023-08-23-12-08-30](https://github.com/alcionai/corso/assets/14259816/387ec727-25d1-4857-85fa-99b307f59a60)